### PR TITLE
Server unit tests

### PR DESCRIPTION
**Pull request by CHelfgott**
My viewpoint on this style issue:
Adding a comma to the end of an array is literally a syntax error in every other language. It will cause your programs to fail. In JS, it is a _choice_, and will work correctly either way.  I will not adhere to a style that will get me into bad habits for every other language I work in if the opposite choice is equally valid. So I am disabling lint at the end of arrays.

### CODE REVIEW

**Comment by klandell** [comment]
@CHelfgott It looks like the tests are blowing up on travis.  Can you take a look?

File: server/routes/api/queries/__mocks__/index.js [hunk 1]
```diff
@@ -0,0 +1 @@
+module.exports = {};
```
Comment (klandell) [inline server/routes/api/queries/__mocks__/index.js]: @CHelfgott Please add the missing newline

Comment (CHelfgott) [reply server/routes/api/queries/__mocks__/index.js]: Done

**Comment by klandell** [comment]
@CHelfgott I'm getting an error message on your branch when I try to run `yarn dev`  Please take a look when you have a moment.

**Comment by klandell** [comment]
@CHelfgott Nvm, I fixed it.

**Pull request merged**